sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/TestPubsubSignal.java

```diff
@@ -242,15 +268,15 @@ private void tearDown() throws IOException {
    * <p>This future must be created before running the pipeline. A subscription must exist prior to
    * the start signal being published, which occurs immediately upon pipeline startup.
    */
-  public Supplier<Void> waitForStart(Duration duration) throws IOException {
+  public Supplier<Void> waitForStart(Duration duration) {
     SubscriptionPath startSubscriptionPath =
         PubsubClient.subscriptionPathFromName(
             pipelineOptions.getProject(),
-            "start-subscription-" + String.valueOf(ThreadLocalRandom.current().nextLong()));
+            "start-subscription-" + ThreadLocalRandom.current().nextLong());
 
     subscriptionAdmin.createSubscription(
-        startTopicPath.getPath(),
         startSubscriptionPath.getPath(),
+        startTopicPath.getPath(),
```

Comment:
Just to double-check - are these edits fine?

Reply:
Yes, method accepts subscription path first and topic path second
There is a similar method invocation with correct arguments later in the class:

https://github.com/akvelon/beam/blob/9cea89e7c2f6622e6f8dfe05624e0be7b7431cc9/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsub/TestPubsubSignal.java#L308-L312